jcr_root/etc/designs/community/sitethemes/bootstrap-custom/clientlibs/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:mixinTypes="[cq:ReplicationStatus]"
     jcr:primaryType="cq:ClientLibraryFolder"
     categories="[cq.social.console.theme.bootstrap-custom]"
-    dependencies="[cq.social.console.theme.scfskin]"/>
+    dependencies="[cq.social.bootstrap.3,cq.social.console.theme.scfskin]"/>
```

Comment:
Hmm, not sure why its required though. It should have been picked up from 

https://github.com/arunpon/scf-custom-theme/blob/master/jcr_root/etc/designs/community/sitethemes/bootstrap-custom/clientlibs/bootstrap-custom.less#L2

Also existing themes dont have that in dependencies /etc/designs/community/sitethemes/bootstrap-journal/clientlibs